pandas/tseries/frequencies.py

```diff
@@ -926,7 +926,8 @@ def _get_wom_rule(self):
             return None
 
         week_of_months = unique((self.index.day - 1) // 7)
-        if len(week_of_months) > 1:
+        week_of_months = week_of_months[week_of_months < 4]
```

Comment:
can you add a comment here on what this does and referene the issue (so future readers will know).